addons/mail/static/src/discuss/call/common/thread_model_patch.js

```diff
@@ -65,11 +72,19 @@ const ThreadPatch = {
                 }
             },
         });
-    },
-    get videoCount() {
-        return Object.values(this.store["discuss.channel.rtc.session"].records).filter(
-            (session) => session.hasVideo
-        ).length;
+        this.videoCount = fields.Attr(0, {
+            compute() {
+                return Object.values(this.store["discuss.channel.rtc.session"].records).filter(
+                    (session) => session.hasVideo
+                ).length;
+            },
```

Comment:
Code was already wrong? It filters on all RTC session that has video without checking the session is on this thread.

I would expect it to use `this.rtc_session_ids.filter((s) => s.hasVideo).length`.